cmake/projects/Boost/atomic/hunter.cmake

```diff
@@ -18,5 +18,5 @@ hunter_download(
     Boost
     PACKAGE_COMPONENT
     atomic
-    PACKAGE_INTERNAL_DEPS_ID "33"
+    PACKAGE_INTERNAL_DEPS_ID "34"
```

Comment:
Actually Boost is not using `hunter_get_toolchain_binaries` module so there is no need to change `PACKAGE_INTERNAL_DEPS_ID`.

Reply:
List of packages to test:
* https://github.com/ruslo/hunter/blob/c6cf6144e1df345ec8ef1dd2b73b67da34fed702/cmake/modules/hunter_autotools_project.cmake#L64-L66

Stack: `hunter_autotools_project` -> `hunter_autotools_configure_command` -> `hunter_get_toolchain_binaries`.

For these packages `PACKAGE_INTERNAL_DEPS_ID` should be updated.

Also please add `libxml2` to this list, it's the only package that cover Android build.